src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs

```diff
@@ -56,7 +56,8 @@ mod tests {
 
     #[test]
     fn failure_if_receipt_result_is_abort() {
-        let encoded_receipt_hex = include_str!("create_pool.dat");
+        let encoded_receipt_hex =
+            include_str!(concat!(env!("FIXTURES_TX"), "create_pool.dat"));
```

Comment:
not great, but not to bad. We must use `concat!` to get a string literal. thx to `env!("FIXTURES_TX")` it is kept relatively short. If we use a shorter name of the env variable it will be shorter, but less clear...